File: CS/RemoveNewButtonInLookups.Module.Win/Module.cs
using System;
using System.Collections.Generic;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;

namespace RemoveNewButtonInLookups.Module.Win {
    public sealed partial class RemoveNewButtonInLookupsWinModule : ModuleBase {
        public RemoveNewButtonInLookupsWinModule() {
            InitializeComponent();
        }
        public override IEnumerable<ModuleUpdater> GetModuleUpdaters(IObjectSpace objectSpace, Version versionFromDB) {
            return ModuleUpdater.EmptyModuleUpdaters;
        }
    }
}

File: VB/RemoveNewButtonInLookups.Module.Win/Module.vb
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating

Namespace RemoveNewButtonInLookups.Module.Win
    Public NotInheritable Partial Class RemoveNewButtonInLookupsWinModule
        Inherits ModuleBase

        Public Sub New()
            InitializeComponent()
        End Sub
        Public Overrides Function GetModuleUpdaters(ByVal objectSpace As IObjectSpace, ByVal versionFromDB As Version) As IEnumerable(Of ModuleUpdater)
            Return ModuleUpdater.EmptyModuleUpdaters
        End Function
    End Class
End Namespace
